swap_meet/item.py

```diff
@@ -1,2 +1,19 @@
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, category = "", condition=0):
```

Comment:
Stay consistent with your syntax.

```suggestion
    def __init__(self, category="", condition=0):
```